From 0c50f7704d2e63cd74abfbf5c5b352a8d7ccb5b9 Mon Sep 17 00:00:00 2001
From: Yu'an Wang <wangyuan46@huawei.com>
Date: Sat, 30 Oct 2021 15:51:32 +0800
Subject: [PATCH] openeuler_defconfig: Enable CONFIG_HW_RANDOM_HISI_GM by
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 0c50f7704d2e63cd74abfbf5c5b352a8d7ccb5b9

 default

driver inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4FHUR

-----------------------------------------------------------

update CONFIG_HW_RANDOM_HISI_V2 to CONFIG_HW_RANDOM_HISI_GM and enable it.
Then add CONFIG_CRYPTO_DEV_HISI_TRNG and enable it.

Signed-off-by: Yu'an Wang <wangyuan46@huawei.com>
Reviewed-by: Weili Qian <qianweili@huawei.com>
Reviewed-by: Longfang Liu <liulongfang@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 0bdb678bff3a..b657941e2836 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -3265,7 +3265,7 @@ CONFIG_HW_RANDOM_TIMERIOMEM=m
 # CONFIG_HW_RANDOM_BA431 is not set
 CONFIG_HW_RANDOM_VIRTIO=m
 CONFIG_HW_RANDOM_HISI=y
-CONFIG_HW_RANDOM_HISI_V2=y
+CONFIG_HW_RANDOM_HISI_GM=m
 CONFIG_HW_RANDOM_XGENE=y
 CONFIG_HW_RANDOM_CAVIUM=y
 # CONFIG_HW_RANDOM_CCTRNG is not set
@@ -6681,6 +6681,7 @@ CONFIG_CRYPTO_DEV_HISI_SEC2=m
 CONFIG_CRYPTO_DEV_HISI_QM=m
 CONFIG_CRYPTO_DEV_HISI_ZIP=m
 CONFIG_CRYPTO_DEV_HISI_HPRE=m
+CONFIG_CRYPTO_DEV_HISI_TRNG=m
 CONFIG_CRYPTO_DEV_HISI_MIGRATION=m
 # CONFIG_CRYPTO_DEV_AMLOGIC_GXL is not set
 CONFIG_ASYMMETRIC_KEY_TYPE=y
-- 
2.26.2

